<?xml version='1.0' encoding='utf-8'?>

<launch>

     <!-- 启动相机节点  -->
    <include file="$(find usb_cam)/launch/usb_cam-test.launch"/>


    <node pkg="hg_desk_robot" name="arm_mqtt_kz" type="arm_mqtt_kz.py" output="screen">
	<param name="mqtt_host" value="192.0.30.132"/>
        <param name="mqtt_port" value="50001"/>
       

    </node>

</launch>
